cmp/grid/GridAutosizeMode.js

```diff
@@ -16,9 +16,14 @@ export const GridAutosizeMode = Object.freeze({
 
     /**
      * Grid will offer affordances such as context menus and gestures to allow users to manually
-     * trigger an autosize only.  Applications should trigger with autosizeAsync().
+     * trigger an autosize only. Applications should trigger with autosizeAsync().
      */
-    ON_DEMAND: 'onDemand'
+    ON_DEMAND: 'onDemand',
+
+    /**
+     * Grid will autosize columns when the GridModel's sizingMode changes.
```

Comment:
We could clarify that onDemand resizing also still supported.

Would `onSizingModeChange` be a bit more clear? I don't think devs will ever need to type it, given that it's the default, so little penalty for length.

Reply:
Anselm and I agree we should make the slight name changer here as well.